### PR TITLE
PorjectPage Update

### PR DESCRIPTION
**Pull request by Houshilian**

### CODE REVIEW

File: src/components/HomePage/HomePage.js [hunk 1]
```diff
@@ -102,8 +102,16 @@ class HomePage extends Component{
...
+      return;
     }
 
+    //display loading on button on submit
+    this.setState({ loading: true });
+
+    setTimeout(() => {
+      this.setState({ loading: false });
+    }, 2000)
+
```
Comment (tiandin) [inline src/components/HomePage/HomePage.js:114]: Does that mean after 2000 seconds, the loading will gone even the api calls are not finished?

**Review by tiandin — COMMENTED** [review]
Left a comment, please resolve

File: src/components/HomePage/HomePage.js (hunk 1 above)
Comment (Houshilian) [reply src/components/HomePage/HomePage.js:114]: no

Comment (tiandin) [reply src/components/HomePage/HomePage.js:114]: Could you explain what are the codes for?

Comment (Houshilian) [reply src/components/HomePage/HomePage.js:114]: Keep the load time at least 2000 milliseconds.

Comment (tiandin) [reply src/components/HomePage/HomePage.js:114]: What will happen if the loading time exceeds 2000 milliseconds? Will the button return to normal?

Comment (Houshilian) [reply src/components/HomePage/HomePage.js:114]: The button will remain loaded for more than 2000 milliseconds only if there is a alert.

Comment (Houshilian) [reply src/components/HomePage/HomePage.js:114]: The button will remain loaded until fetching data is complete.

**Review by tiandin — APPROVED** [review]
LGTM

**Pull request merged**